src/auth/models/jwt-payload-rt.type.ts

```diff
@@ -0,0 +1,3 @@
+import { JwtPayload } from './jwt-payload.type';
+
+export type JwtPayloadWithRt = JwtPayload & { refreshToken: string };
```

Comment:
hmm, I wonder if we want to have a refresh token in JWT payload? It would be kind of an inception - JWT token inside JWT token 😄